WordPress/Classes/ViewRelated/Likes/LikesListController.swift

```diff
@@ -0,0 +1,343 @@
+import Foundation
+import UIKit
+import WordPressKit
+
+
+/// Convenience class that manages the data and display logic for likes.
+/// This is intended to be used as replacement for table view delegate and data source.
+class LikesListController: NSObject {
+
+    private let formatter = FormattableContentFormatter()
+
+    private let dependency: LikesListDependency
+
+    private let content: ContentIdentifier
+
+    private let siteID: NSNumber
+
+    private let notification: Notification?
+
+    private let tableView: UITableView
+
+    private var likingUsers: [RemoteUser] = []
+
+    private weak var delegate: LikesListControllerDelegate?
+
+    private var isLoadingContent: Bool = false {
+        didSet {
+            isLoadingContent ? activityIndicator.startAnimating() : activityIndicator.stopAnimating()
+            tableView.reloadData()
+        }
+    }
+
+    private var isShowingError: Bool = false
+
+    // MARK: Views
+
+    private lazy var activityIndicator: UIActivityIndicatorView = {
+        let view = UIActivityIndicatorView(style: .medium)
+        view.translatesAutoresizingMaskIntoConstraints = false
+
+        return view
+    }()
+
+    private lazy var loadingCell: UITableViewCell = {
+        let cell = UITableViewCell()
+
+        cell.addSubview(activityIndicator)
+        NSLayoutConstraint.activate([
+            activityIndicator.safeCenterXAnchor.constraint(equalTo: cell.safeCenterXAnchor),
+            activityIndicator.safeCenterYAnchor.constraint(equalTo: cell.safeCenterYAnchor)
+        ])
+
+        return cell
+    }()
+
+    /**
+     TODO: Update this once we get more updates from the design?
```

Comment:
Yea, we would do something different here. Can you undo the `errorCell` change please? It'll just have to be removed later. If we do it now while it's fresh, there won't be any confusion later. Thanks!

Reply:
Oh alright, noted on this.

Reply:
Alright, noted! Sorry for this. I think I misunderstood the comment on previous review. 🙏

Reply:
Oh! Sorry to confuse! Thank you!